src/Elmish.WPF/ViewModel.fs

```diff
@@ -261,7 +262,7 @@ and internal FuncsFromSubModelSeqKeyed() =
 and internal Initialize
       (loggingArgs: LoggingViewModelArgs,
        name: string,
-       getFunctionsForSubModelSelectedItem: string -> ((obj -> obj) * (obj -> ViewModel<obj, obj> option)) option) =
+       getFunctionsForSubModelSelectedItem: string -> ((obj -> obj) * (obj -> obj option) * (obj -> obj)) option) =
```

Comment:
I feel like this could use a record. It's ultimately the following fields from `SubModelSelectedItemBinding<'model, 'msg, 'bindingModel, 'bindingMsg, 'bindingViewModel, 'id>`:
```fsharp
{ GetId: 'bindingModel -> 'id
  FromId: 'id -> 'bindingViewModel option
  GetUnderlyingModel: 'bindingViewModel -> 'bindingModel }
```

Reply:
Yes, that sounds like a good idea

Reply:
I'll make that as a separate PR. I already have a commit for it. I think it needs to go after this PR.

Reply:
@TysonMN #454